docs/source/technical_tutorials/working_with_r/vector data visualization.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    file_path &lt;- "/projects/data/GEDI04_A_2019107224731_O01958_01_T02638_02_002_02_V002.h5"
</pre><p>Why is the file hard coded here instead of passed from a variable from above where it was <code>download_file</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>